packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -129,10 +129,53 @@ export class PHPRequestHandler implements RequestHandler {
 			this.rewriteRules
 		);
 		const fsPath = `${this.#DOCROOT}${normalizedRequestedPath}`;
-		if (seemsLikeAPHPRequestHandlerPath(fsPath)) {
-			return await this.#dispatchToPHP(request, requestedUrl);
+		if (this.#isStaticFile(fsPath)) {
+			return this.#serveStaticFile(fsPath);
+		} else if (this.#isStaticWpCoreFile(fsPath)) {
```

Comment:
We shouldn't have WordPress-specific code in PHP-wasm but I'm not sure how to resolve this.

I think that the best solution would be to [catch this in service-worker](https://github.com/WordPress/wordpress-playground/blob/c988f0607058b84b73206037e87ebda271b2dbf5/packages/playground/remote/service-worker.ts#L26) and not even trigger the request handler.

The problem is that service-worker can't check if the file exists, so we would need to have a list of available core files.